src/lib.ts

```diff
@@ -35,17 +36,24 @@ ${fence}
 }
 
 export async function parse(
-  url: string,
+  link: string,
   turndownService: TurndownService,
   format: string,
 ): Promise<Article> {
-  const response = await axios.get(url);
-  const doc = new JSDOM(response.data);
+  let rawData = "";
+  // check the link is a local file or not
+  if(fs.existsSync(link)) {
+    // I think I going to add a encoding option: --encoding=utf8
```

Comment:
saw 'utf8' as the parameter already. Can you remove the comment?

Reply:
I know you might not like it, but that can remind you that there could be issues in the future. 
You need to add another optional parameter for that.
And I already allow edits by maintainers.

![Screenshot 2023-08-27 at 8 46 55 AM](https://github.com/aome510/article-md-cli/assets/20425883/25cee001-e8d7-400d-9e1e-72f017fb8480)

Reply:
oh so you mean the optional cli option, I thought passing 'utf8' to the function call below. Just want to make sure we're on the same page. 'utf8' by default should work fine. Can always add a cli option later if it doesn't work for someone else.

Reply:
You are right.
I mean the cli options.